### ISSUE TITLE
Help: about config file

### ISSUE THREAD

**Issue by fdorazio** [qdm12/ddns-updater#441]
**TLDR**: *Hi, 
I have a problem with my json config file.

When I deploy ddns-updater container, I obtain this error message:

2023-02-24T07:54:54Z ERROR configuration given: cannot unmarshal common settings: invalid character '/' looking for beginning of value

I try too with a config to noip, I have the same issue.

Can you help me please ?

Thanks 

Have a nice day.
*

1. Is this urgent: Yes and no, it's possible to help me a little quick ?
2. DNS provider(s) you use: test done with noip and duckdns
3. Program version:

    

    `Running version latest built on 2023-02-02T00:41:38Z (commit 228fc50)`

4. What are you using to run the container: I use Kubernetes in v1.26
5. Extra information (optional)

Logs:

```
========================================
========================================
============= ddns-updater =============
========================================
=========== Made with ❤️ by ============
======= https://github.com/qdm12 =======
========================================
========================================

Running version latest built on 2023-02-02T00:41:38Z (commit 228fc50)

🔧 Need help? https://github.com/qdm12/ddns-updater/discussions/new
🐛 Bug? https://github.com/qdm12/ddns-updater/issues/new
✨ New feature? https://github.com/qdm12/ddns-updater/issues/new
☕ Discussion? https://github.com/qdm12/ddns-updater/discussions/new
💻 Email? quentin.mcgaw@gmail.com
💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
2023-02-24T07:54:54Z INFO reading JSON config from environment variable CONFIG
2023-02-24T07:54:54Z DEBUG config read: /updater/data/config.json
2023-02-24T07:54:54Z ERROR configuration given: cannot unmarshal common settings: invalid character '/' looking for beginning of value
2023-02-24T07:54:54Z INFO Shutdown successful

```

Configuration file (**remove your credentials!**):

```json
{
        "settings": [
                {
                        "provider": "duckdns",
                        "domain": "mysubdomain",
                        "token": "*****MyDuckDNS Token*****"
                }
        ]
}
```

Host OS:
Centos 9 with kubernetes 1.26

In advance thanks a lot.

**Comment by fdorazio** [comment]
Hi @sebastian-muthwill,

I changed my configuration from config map to file in pvc.
And I added more informations below token variable:

      "ip_version": "ipv4",
      "provider_ip": true

I hope that it will help you.

Have a nice day.
